### PR TITLE
feat: enter code step more styling

### PR DESCRIPTION
**Pull request by danielgent**

### CODE REVIEW

File: src/styles/tailwind.ts
Comment (danielgent) [inline src/styles/tailwind.ts:1]: I'll do another PR updating the snapshots once this is released

File: src/utils/components/LoginEnterCodePage.tsx [hunk 1]
```diff
@@ -33,21 +37,47 @@ const LoginEnterCodePage: FC<Props> = ({ error, vendorSettings, email }) => {
           values={{ email }}
         /> */}
         {i18next
-          .t("we_sent_a_code_to", { email })
+          .t("we_sent_a_code_to", {
+            // email: `<span class="text-black dark:text-white">${email}</span>`,
```
Comment (danielgent) [inline src/utils/components/LoginEnterCodePage.tsx:41]: :joy:  I just want the email in bold like on login2 :joy:

File: src/styles/tailwind.ts [hunk 2]
```diff
@@ -801,6 +801,10 @@ h6 {
 .relative {
   position: relative;
 }
+.my-4 {
+  margin-top: 1rem;
+  margin-bottom: 1rem;
+}
```
Comment (danielgent) [inline src/styles/tailwind.ts:807]: I'll do another PR updating the snapshots once this is released. It's probably worth coming up with a solution where we can serve the latest CSS locally in the CI/CD...

File: test/integration/login/__image_snapshots__/code-flow-liquidjs-spec-ts-test-integration-login-code-flow-liquidjs-spec-ts-login-with-code-on-liquidjs-template-should-login-with-code-2-snap.png
Comment (danielgent) [inline test/integration/login/__image_snapshots__/code-flow-liquidjs-spec-ts-test-integration-login-code-flow-liquidjs-spec-ts-login-with-code-on-liquidjs-template-should-login-with-code-2-snap.png:1]: this looks much better when ran locally :see_no_evil:

**Review by markusahlstrand — APPROVED** [review]
LGTM

**Pull request merged**